JS/auth0.js

```diff
@@ -0,0 +1,33 @@
+const { AUTH0_SECRET, AUTH0_CLIENT_ID, AUTH0_ISSUER_BASE_URL } = process.env;
+const port = process.env.port || 3000;
+let auth_config = undefined;
+
+const secretGiven = () => AUTH0_SECRET != undefined;
+const idGiven = () => AUTH0_CLIENT_ID != undefined;
+const isValidUrl = () => {
+    try {
+      new URL(AUTH0_ISSUER_BASE_URL);
+    } catch (_) {
+      return false;  
+    }
+    if (string.startsWith("https://")) {
+        return true
+    } else {
+        throw new Error("Auth0 issuer base url is not valid, must begin with 'https://'");
+    }
+}
+
+if (secretGiven && idGiven && isValidUrl) {
```

Comment:
FYI I think this validation also didn't run because you just checked for the truthiness of the function, not calling it. But since I don't think we want it, we can just remove it anyway.